sunpy/timeseries/metadata.py

```diff
@@ -519,8 +520,8 @@ def _validate_meta(self, meta):
                 col_overlap = list(set(self.metadata[i][1]) & set(self.metadata[j][1]))
                 # If we have an overlap then show a warning
                 if col_overlap:
-                    warnings.warn_explicit('Metadata entries ' + str(i) + ' and ' + str(j) + ' contain interleaved data.',
-                                           Warning, __file__, inspect.currentframe().f_back.f_lineno)
+                    warnings.warn('Metadata entries {0} and {1} contain interleaved data.'.format(str(i), str(j)),
```

Comment:
You can just use f-strings
```suggestion
                    warnings.warn(f'Metadata entries {i} and {j} contain interleaved data.',
```